alfasim_sdk_api/detail/api_pointers.h

```diff
@@ -7,14 +7,15 @@ set_plugin_data_func set_plugin_data;
 
```

Comment:
IMO, API documentation should be moved to `api.h`, instead of leaving them on this detail header file, since the user shoudn't have to bother about the details. What do you guys think? @williamjamir @ggrbill

Reply:
Agreed. It seems that things are hidden here.

Reply:
Yes, but when using the IDE, the "follow the implementation" or whatever other names it may be, brings the developer to this file.

Reply:
Ah... Ok. Thanks for pointing this out again.